README.md

```diff
@@ -1,18 +1,45 @@
 
+## Documentation 
+uBLAS is documented at [boost.org](https://www.boost.org/doc/libs/1_69_0/libs/numeric/ublas/doc/index.html).
+The development has a [wiki page](https://github.com/uBLAS/ublas/wiki).
+The tensor extension has a separate [wiki page](https://github.com/BoostGSoC18/tensor/wiki).
 
-The Boost.uBLAS Linear Algebra Library v1.0
+## License
+Distributed under the [Boost Software License, Version 1.0](http://www.boost.org/LICENSE_1_0.txt).
 
-- To follow development and test experimental features, you can clone the Github project uBLAS/ublas
-at https://github.com/uBLAS/ublas
-- A development wiki is available at https://github.com/uBLAS/ublas/wiki
-- A mailing-list is available at http://lists.boost.org/ublas/
-- For any other questions, you can contact David at david.bellot@gmail.com
+## Properties
+* Header-only
+* Tensor extension requires C++17 compatible compiler, compiles with
+  * gcc 7.3.0
+  * clang 6.0
+  * msvc 14.1
+* Unit-tests require Boost.Test
 
-- version numbers have never been used for this library until 02 March 2014.
-  So we start at v1.0 on that day.
+## Build Status
 
-last update: 1 April 2014
+Branch          | Travis | Appveyor | codecov.io | Docs |
+:-------------: | ------ | -------- | ---------- | ---- |
+[`master`](https://github.com/boostorg/ublas/tree/master) | [![Build Status](https://travis-ci.org/boostorg/ublas.svg?branch=master)](https://travis-ci.org/boostorg/ublas) | [![Build status](https://ci.appveyor.com/api/projects/status/ctu3wnfowa627ful/branch/master?svg=true)](https://ci.appveyor.com/project/stefanseefeld/ublas/branch/master) | [![codecov](https://codecov.io/gh/boostorg/ublas/branch/master/graph/badge.svg)](https://codecov.io/gh/boostorg/ublas/branch/master) | [![Documentation](https://img.shields.io/badge/docs-develop-brightgreen.svg)](http://www.boost.org/doc/libs/release/libs/numeric)
+[`develop`](https://github.com/boostorg/ublas/tree/develop) | [![Build Status](https://travis-ci.org/boostorg/ublas.svg?branch=develop)](https://travis-ci.org/boostorg/ublas) | [![Build status](https://ci.appveyor.com/api/projects/status/ctu3wnfowa627ful/branch/develop?svg=true)](https://ci.appveyor.com/project/stefanseefeld/ublas/branch/develop) | [![codecov](https://codecov.io/gh/boostorg/ublas/branch/develop/graph/badge.svg)](https://codecov.io/gh/boostorg/ublas/branch/develop) | [![Documentation](https://img.shields.io/badge/docs-develop-brightgreen.svg)](http://www.boost.org/doc/libs/release/libs/numeric)
+
+
+## Directories
+
+| Name        | Purpose                        |
+| ----------- | ------------------------------ |
+| `doc`       | documentation                  |
+| `examples`  | example files                  |
+| `include`   | headers                        |
+| `test`      | unit tests                     |
```

Comment:
Would you mind adding the `benchmarks` directory to the list ?

Reply:
of course not.